packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -1,160 +1,142 @@
-    "description": "The tooltip for the MonthPicker's 'next month' button.",
-    "type": "text"
+    "description": "The tooltip for the MonthPicker's 'next month' button."
   },
 
   "previousMonthTooltip": "Previous month",
   "@previousMonthTooltip": {
-    "description": "The tooltip for the MonthPicker's 'previous month' button.",
-    "type": "text"
+    "description": "The tooltip for the MonthPicker's 'previous month' button."
   },
 
   "nextPageTooltip": "Next page",
-  "@nextMonthTooltip": {
-    "description": "The tooltip for the [PaginatedDataTables]'s 'next page' button.",
-    "type": "text"
+  "@nextPageTooltip": {
+    "description": "The tooltip for the [PaginatedDataTables]'s 'next page' button."
   },
 
   "previousPageTooltip": "Previous page",
-  "@previousMonthTooltip": {
-    "description": "The tooltip for the PaginatedDataTables's 'previous page' button.",
-    "type": "text"
+  "@previousPageTooltip": {
+    "description": "The tooltip for the PaginatedDataTables's 'previous page' button."
   },
 
   "showMenuTooltip": "Show menu",
   "@showMenuTooltip": {
-    "description": "The default PopupMenuButton tooltip",
-    "type": "text"
+    "description": "The default PopupMenuButton tooltip"
   },
 
   "aboutListTileTitle": "About $applicationName",
   "@aboutListTileTitle": {
     "description": "The default title for AboutListTile",
-    "type": "text"
+    "parameters": "applicationName"
   },
 
   "licensesPageTitle": "Licenses",
   "@licensesPageTitle": {
-    "description": "The title for the Flutter licenses page.",
-    "type": "text"
+    "description": "The title for the Flutter licenses page."
   },
 
   "pageRowsInfoTitle": "$firstRow–$lastRow of $rowCount",
-  "pageRowsInfoTitleApproximate": "$firstRow–$lastRow of about $rowCount",
   "@pageRowsInfoTitle": {
-    "description": "Title for the [PaginatedDataTable]'s row info footer",
-    "type": "text"
+    "description": "Title for the [PaginatedDataTable]'s row info footer when the exact overall row count is known",
+    "parameters": "firstRow, lastRow, rowCount"
+  },
+
+  "pageRowsInfoTitleApproximate": "$firstRow–$lastRow of about $rowCount",
+  "@pageRowsInfoTitleApproximate": {
+    "description": "Title for the [PaginatedDataTable]'s row info footer when the exact overall row count is unknown",
+    "parameters": "firstRow, lastRow, rowCount"
   },
 
   "rowsPerPageTitle": "Rows per page:",
   "@rowsPerPageTitle": {
-    "description": "Title for the [PaginatedDataTable]'s 'rows per page' footer.",
-    "type": "text"
+    "description": "Title for the [PaginatedDataTable]'s 'rows per page' footer."
   },
 
   "selectedRowCountTitleZero": "No items selected",
   "selectedRowCountTitleOne": "1 item selected",
   "selectedRowCountTitleOther": "$selectedRowCount items selected",
   "@selectedRowCountTitle": {
     "description": "Title for the PaginatedDataTable's selected row count header",
-    "type": "text"
+    "plural": "selectedRowCount"
```

Comment:
Should this be:

```json
"type": "plural",
"parameters": "selectedRowCount"
```

?

Reply:
If we end up need a plural translation that has additional parameters one would specify both:
```
"plural": "quantityVar",
"parameters": "otherParameter1, otherParameter2"
```
For now we don't need that.